llvm/lib/ObjCopy/COFF/COFFObject.cpp

```diff
@@ -16,8 +16,11 @@ namespace coff {
 using namespace object;
 
 void Object::addSymbols(ArrayRef<Symbol> NewSymbols) {
+  size_t RawIndex = 0;
```

Comment:
@mstorsjo I've noticed that this approach is error-prone as if `addSymbols` invoked twice, symbols will have duplicated `OriginalRawIndex` (It doesn't happen now). Looks like we should have this variable as a class member.